gui/wxpython/gmodeler/model.py

```diff
@@ -669,7 +669,7 @@ def Run(self, log, onDone, parent=None):
                     parent=parent,
                     message=_("Variables below not defined:") +
                     "\n\n" +
-                    unicode(
+                    grass.decode(
```

Comment:
There seems to be no need to encode/decode. `err` contains tuple of strings.

Reply:
Alright of course, I remove this function.